tests/distributed/test_fsdp_optim_state.py

```diff
@@ -0,0 +1,165 @@
+import copy
+
+import torch
+import torch.distributed as dist
+import torchacc as ta
+from torchacc.dist.fsdp import FullyShardedDataParallel as FSDP
+from torchacc.config import Config
+
+from utils.distributed import MultiProcessTestBase, init_pg, skip_if_lt_x_gpu
+
+
+class Net(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+        self.fc1 = torch.nn.Linear(1024, 1024)
+        self.fc2 = torch.nn.Linear(1024, 1024)
+        self.fc3 = torch.nn.Linear(1024, 1024)
+        self.fc4 = torch.nn.Linear(1024, 1024)
+        self.fc5 = torch.nn.Linear(1024, 1024)
+
+    def forward(self, x):
+        x = self.fc1(x)
+        x = self.fc2(x)
+        x = self.fc3(x)
+        x = self.fc4(x)
+        x = self.fc5(x)
+        return x
+
+
+def _init_model(config: Config):
+    model = Net()
+    model = FSDP(model, config)
+    optim = torch.optim.AdamW(model.parameters(), lr=0.1)
+
+    return model, optim
+
+
+def _step_model(
+    model: torch.nn.Module,
+    optim: torch.optim.Optimizer,
+    num_iters: int = 1,
+):
+    torch.manual_seed(0)  # set seed for determinism
```

Comment:
Setting a seed at the beginning of the UT, it is not necessary to set the seed every time.

Reply:
done